src/main/java/quacky/Event.java

```diff
@@ -0,0 +1,40 @@
+package quacky;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event task in the Quacky application. An event task is a task
+ * that occurs over a period of time, from a start date to an end date.
+ */
+public class Event extends Task{
+    protected LocalDate from;
```

Comment:
Perhaps `fromDate` would sound more like a noun for this variable? To obey code quality guidelines.